docs/contributor/performance-tests/metrics/metrics.md

```diff
@@ -0,0 +1,808 @@
+# Hardened Metric Agent setup
+
+The aim of this exercise is to harden the metric agent so that it can satisfy the metric load of most of the use cases.
+
+## Setup
+
+For the test environment following setup used:
```

Comment:
Thanks for reworking this, but please also use the [suggested grammar](https://github.com/kyma-project/telemetry-manager/pull/406#discussion_r1328704406).

```suggestion
For the test environment, the following setup was used:
```